### ISSUE TITLE
What does frequency mean?

### ISSUE THREAD

**Issue by Gurmukh-Singh-4253** [Waradu/Mond#4]
I have a time tab in my sketchybar config as well, which refreshes every 5 seconds. 
Mond on the other hand, refreshes atleast once every second. So I tried tinkering with the refreshFrequency variable to make it also refresh once every 5 seconds but was unsuccessful, because I don't know how we are measuring the frequency. Is it the refreshes per second or the time delay in each refresh?

**Comment by Waradu** [comment]
refreshFrequency: 100
"It defines the delay in milliseconds between consecutive commands executions." like stated in the Übersicht [GitHub](https://github.com/felixhageloh/uebersicht#refreshfrequency)
This means the widget refreshes every "refreshFrequency" seconds.
The Mond widget refreshes every 0.1 seconds, which is equivalent to 100 milliseconds. Setting it to 5000 would mean a refresh every 5 seconds, but note that this setting may cause issues with the seconds display.

**Comment by Gurmukh-Singh-4253** [comment]
yes but it seems to add another issue, the application seems to stop working post suspension. can you please tell how to solve this issue
